File: Terceiro semestre/java/sistema_login_java/Usuario.java
public class Usuario {
    public String email;
    public String usuario;
    public String senha;
    public String telefone;
    public String idade;
    public String genero;
    public String foto_de_perfil;

    public Usuario() {}

    public Usuario(String email, String usuario, String senha, String telefone, String idade, String genero, String foto) {
        this.email = email;
        this.usuario = usuario;
        this.senha = senha;
        this.telefone = telefone;
        this.idade = idade;
        this.genero = genero;
        this.foto_de_perfil = foto;
    }
}

File: Terceiro semestre/python/sistema_login_python/usuario.py
class Usuario:
    def __init__(self,email,usuario,senha,telefone,idade,genero,foto_de_perfil):
        self.email = email
        self.usuario = usuario
        self.senha = senha
        self.telefone = telefone
        self.idade = idade
        self.genero = genero
        self.foto_de_perfil = foto_de_perfil
    def to_dict(self):
        return self.__dict__
    